src/vs/workbench/contrib/issue/browser/issueFormService.ts

```diff
@@ -0,0 +1,180 @@
+		if (data.extensionId && this.extensionIdentifierSet.has(data.extensionId)) {
+			const replyChannel = `vscode:triggerReporterMenuResponse`;
+			mainWindow.postMessage({ data, replyChannel }, '*');
+			this.extensionIdentifierSet.delete(new ExtensionIdentifier(data.extensionId));
+		}
+
+		if (this.issueReporterWindow) {
+			this.issueReporterWindow.focus();
+			return;
+		}
+
+		const disposables = new DisposableStore();
+
+		// Auxiliary Window
+		const auxiliaryWindow = disposables.add(await this.auxiliaryWindowService.open({ mode: AuxiliaryWindowMode.Normal }));
+
+		this.issueReporterWindow = auxiliaryWindow.window;
+
+
+
+		if (auxiliaryWindow) {
+			await auxiliaryWindow.whenStylesHaveLoaded;
+			auxiliaryWindow.window.document.title = 'Issue Reporter';
+			auxiliaryWindow.window.document.body.classList.add('issue-reporter-body');
+
+			// custom issue reporter wrapper
+			const div = document.createElement('div');
+			div.classList.add('monaco-workbench');
+
+			// removes preset monaco-workbench
+			auxiliaryWindow.container.remove();
+			auxiliaryWindow.window.document.body.appendChild(div);
+			safeInnerHtml(div, BaseHtml());
+
+			// create issue reporter and instantiate
+			const issueReporter = this.instantiationService.createInstance(IssueWebReporter, false, data, { type: '', arch: '', release: '' }, product, auxiliaryWindow.window);
+			issueReporter.render();
+		} else {
+			console.error('Failed to open auxiliary window');
+		}
+
+		// handle closing issue reporter
+		this.issueReporterWindow?.addEventListener('beforeunload', () => {
+			auxiliaryWindow.window.close();
+			this.issueReporterWindow = null;
+		});
+	}
+
+	async openProcessExplorer(data: ProcessExplorerData): Promise<void> {
+		throw new Error('Method not implemented.');
+	}
+
+	stopTracing(): Promise<void> {
+		throw new Error('Method not implemented.');
+	}
+	getSystemStatus(): Promise<string> {
+		throw new Error('Method not implemented.');
+	}
+	$getSystemInfo(): Promise<SystemInfo> {
+		throw new Error('Method not implemented.');
+	}
+	$getPerformanceInfo(): Promise<PerformanceInfo> {
+		throw new Error('Method not implemented.');
+	}
+	$reloadWithExtensionsDisabled(): Promise<void> {
+		throw new Error('Method not implemented.');
+	}
+	$showConfirmCloseDialog(): Promise<void> {
+		throw new Error('Method not implemented.');
+	}
+	$showClipboardDialog(): Promise<boolean> {
+		throw new Error('Method not implemented.');
+	}
+	$getIssueReporterUri(extensionId: string): Promise<URI> {
+		throw new Error('Method not implemented.');
+	}
+	$getIssueReporterData(extensionId: string): Promise<string> {
+		throw new Error('Method not implemented.');
+	}
+	$getIssueReporterTemplate(extensionId: string): Promise<string> {
+		throw new Error('Method not implemented.');
+	}
+	$getReporterStatus(extensionId: string, extensionName: string): Promise<boolean[]> {
+		throw new Error('Method not implemented.');
+	}
+
```

Comment:
Are these going to be implemented in the future?

Reply:
it's hard to say, the interface might change because the return values (ie, `Promise<PerformanceInfo>` or `Promise<SystemInfo>`) are electron specific and system info is just handled elsewhere. 

the end goal is to use the aux window for the original issue report window as well, and I'll likely change and modify the interface then.